Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36871&attempt_id=e4fd99ae-92d1-4b0f-8a92-c3ba6f54ab1b)

Attempted: clingo

<details><summary>Partial log (click to expand)</summary><p>

```
[ 92%] Building CXX object libpyclingo/CMakeFiles/libpyclingo.dir/pyclingo.cc.o
/tmp/nix-build-clingo-5.2.2.drv-0/source/libpyclingo/pyclingo.cc:32:10: fatal error: 'Python.h' file not found
#include <Python.h>
         ^~~~~~~~~~
1 error generated.
make[2]: *** [libpyclingo/CMakeFiles/libpyclingo.dir/build.make:63: libpyclingo/CMakeFiles/libpyclingo.dir/pyclingo.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:521: libpyclingo/CMakeFiles/libpyclingo.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
builder for '/nix/store/6ssnz1bjxh964l9s3z6d20fc9w2x335r-clingo-5.2.2.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/6ssnz1bjxh964l9s3z6d20fc9w2x335r-clingo-5.2.2.drv' failed
```
</p></details>

--- a/pkgs/applications/science/logic/potassco/clingo.nix
+++ b/pkgs/applications/science/logic/potassco/clingo.nix
@@ -13,6 +13,8 @@ stdenv.mkDerivation rec {
   buildInputs = [];
   nativeBuildInputs = [cmake];
 
+  cmakeFlags = [ "-DCLINGO_BUILD_WITH_PYTHON=OFF" ];
+
   meta = {
     inherit version;
     description = "ASP system to ground and solve logic programs";

